bwa-sys/build.rs

```diff
@@ -39,9 +38,10 @@ fn main() {
     }
     cc::Build::new()
         .define("COMPILATION_TIME_PLACE", "\"build.rs\"")
+        .warnings(false)
+        .extra_warnings(false)
         .files(FILES)
         .flag("-g")
```

Comment:
I wouldn't include this here.  Under the bazel build, that will come in via `CFLAGS` for the appropriate build modes.  Same goes for `-O2`.  I would, however, consider adding
```suggestion
        .flag("-std=c++11")
```
or possibly even higher level standards like `c++14` or `c++17` as that can actually improve performance quite noticeably, assuming it still compiles properly, by enabling move constructors and copy-on-write strings.

Reply:
This is C code and doesn't compile with c++11. I dropped the flags.